### PR TITLE
Http redirect

### PR DESCRIPTION
**Pull request by claudiaop**

### CODE REVIEW

File: JMS/activemq/docker-compose.5.10.0.yml [hunk 1]
```diff
@@ -2,8 +2,11 @@ version: '2'
 services:
   gateway:
     build: gateway
+    environment: 
+      - GATEWAY_FEATURES=http.redirect
```
Comment (dpwspoon) [inline JMS/activemq/docker-compose.5.10.0.yml]: Let's wait until this is no longer an EA flag before releasing

File: JMS/activemq/docker-compose.5.10.0.yml [hunk 2]
```diff
@@ -2,8 +2,11 @@ version: '2'
...
     ports:
       - "8000:8000"
```
Comment (dpwspoon) [inline JMS/activemq/docker-compose.5.10.0.yml]: The service ports should go to 443.
The redirect service should be on port 80

**Comment by dpwspoon** [comment]
Thanks @claudiaop.  Let's hold off on merging this until the feature is released

File: JMS/activemq/docker-compose.5.10.0.yml (hunk 1 above)
Comment (claudiaop) [inline JMS/activemq/docker-compose.5.10.0.yml]: ok

File: JMS/activemq/docker-compose.5.10.0.yml (hunk 2 above)
Comment (claudiaop) [inline JMS/activemq/docker-compose.5.10.0.yml]: i've changed them

**Pull request closed without merging**